.env.example

```diff
@@ -3,3 +3,5 @@ SSR_PORT=3030
 MEDIAWIKI_NETWORK_TO_JOIN=
 CSR_PORT=8080
 NODE_ENV=development
+# set to e.g. the api script of your development environment's mediawiki
+WIKIBASE_REPO_API=http://default.web.mw.localhost/mediawiki/api.php
```

Comment:
@jakobw Thanks for poking me on this. Do you think the version w/ or w/o the /mediawiki path is more common?

Reply:
I think with /mediawiki is. That's the one that works for me, too.